src/components/AccountDropdown/AccountDropdown.tsx

```diff
@@ -40,12 +46,25 @@ import { AccountSegment } from './AccountSegement'
 
 export type AccountDropdownProps = {
   assetId: AssetId
-  buttonProps?: ButtonProps
   onChange: (accountId: AccountId) => void
+  accountId?: AccountId
+  // Auto-selects the account with the highest balance, and sorts the account list descending by balance
+  autoSelectHighestBalance?: boolean
+  // Prevents accounts in the dropdown from being selected
+  disableSelection?: boolean
```

Comment:
Would prefer `disabled` to honor the [`disabled` HTML attribute](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/disabled)

Reply:
Checks out, actioned: https://github.com/shapeshift/web/pull/2684/commits/46ed0f962c5c2a76b6d04f1f22f075b75c602264